app/controllers/api/v1/users_controller.rb

```diff
@@ -5,6 +5,10 @@ module V1
     class UsersController < ApiController
       skip_before_action :ensure_authenticated, only: %i[create]
 
+      before_action only: %i[update destroy purge_avatar] do
```

Comment:
`before_action :ensure_authorized('ManageUsers`), only: %i[update destroy purge_avatar]

Reply:
Can't pass an arg to :ensure_authorized, it needs to be in a block.